components/fieldgroups/campaign-member/campaign-member-components.schema.json

```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/campaign-member"],
-  "description": "This mixin is used to capture B2B campaign member specific relationship attributes that are represented as it is in source and also used for merge/unmerge rules analysis if and when needed.",
+  "description": "This field group is used to capture B2B campaign member specific relationship attributes that are represented as it is in source and also used for merge/unmerge rules analysis if and when needed.",
```

Comment:
"Capture B2B campaign member specific relationship attributes that are represented as it is in source and also us..."

Make the description as short as possible so users can see the purpose of the field group quickly